CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
-cmake_minimum_required(VERSION 2.6.4)
+cmake_minimum_required(VERSION 3.1)
 
 option(WITH_TESTS           "Build tests."       OFF)
 option(WITH_EXAMPLE         "Build example."     OFF)
 option(WITH_STATIC          "Build static libs." ON)
-option(WITH_CPP11           "Build with C++11."  ON)
-option(WITH_CPP14           "Build with C++14."  OFF)
```

Comment:
I see, Travis has only CMake 2.8.
CMake 3.1 supports setting C++ version using CMAKE_CXX_STANDARD variable. It is better, because you can set this variable in parent project, and then include "docopt" directory in it (and link it using link_library()).

I will rewrite travis.yml to update CMake version (to version 3.2.2 for example, as used in Ubuntu Wily) before building the code. Do not close this pull request, please.